File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/control/controllers/gains/FeedforwardGains.java
package org.firstinspires.ftc.teamcode.control.controllers.gains;

public class FeedforwardGains {

    public final double kV, kA, kStatic;

    public FeedforwardGains(
            double kV,
            double kA,
            double kStatic
    ) {
        this.kV = kV;
        this.kA = kA;
        this.kStatic = kStatic;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/control/controllers/coefficients/FeedforwardGains.kt
package org.firstinspires.ftc.teamcode.control.controllers.coefficients

data class FeedforwardGains @JvmOverloads constructor(
    val kV: Double = 0.0,
    val kA: Double = 0.0,
    val kStatic: Double = 0.0
)
